README.md

```diff
@@ -65,7 +65,10 @@ curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microso
 sudo install -o root -g root -m 644 microsoft.gpg /usr/share/keyrings/microsoft-archive-keyring.gpg
 sudo sh -c 'echo "deb [arch=amd64,arm64,armhf signed-by=/usr/share/keyrings/microsoft-archive-keyring.gpg] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
 sudo apt update
-sudo apt install code gdb-multiarch ccache clang clangd llvm lld libguestfs-tools libssl-dev trace-cmd python3-pip jsonnet libelf-dev bison bindfs mmdebstrap proot systemtap flex yacc bc debian-archive-keyring
+sudo apt install code gdb-multiarch ccache clang clangd llvm lld libguestfs-tools libssl-dev trace-cmd python3-pip jsonnet libelf-dev bison bindfs mmdebstrap proot systemtap flex yacc bc debian-archive-keyring qemu-system-arm crossbuild-essential-arm64 qemu-user-static binfmt-support
```

Comment:
The QEMU changes here look legit but that crossbuild-essential-arm64 seems fishy. This pulls in GCC and g++. This setup intentionally sticks to LLVM because it has further implications like the clangd VSCode extension working properly to provide LSP etc... It looks like you end up on a frankenstein GCC/LLVM setup which will be half broken.

Reply:
Only the `CROSS_COMPILE` bits in `tasks.sh` and the `crossbuild-essential-arm64` package here seem unrelated but installing `qemu-system-arm qemu-user-static binfmt-support qemu-user-binfmt` would still make sense, feel free to re-open a PR for that if you'd like to contribute those bits.